source/aws-bootstrap-kit/test/aws-organizations-stack.test.ts

```diff
@@ -250,6 +253,20 @@ test("should not create any zone if no domain is provided", () => {
   expect(awsOrganizationsStack).toCountResources("AWS::Route53::HostedZone",0);
```

Comment:
not sure to understand why we have 0 in that case?

Reply:
if you don't give a domain we assume you don't want to get DNS management in Bootstrap Kit

Reply:
sorry, misread because of github collapses...